src/main/java/org/jitsi/jigasi/TranscriptionGateway.java

```diff
@@ -133,24 +140,36 @@ private String getCustomTranscriptionServiceClass(String tenant)
                     }
                     JSONObject obj = new JSONObject(responseBody.toString());
                     transcriberClass = obj.getString("transcriber");
-                    if (logger.isDebugEnabled())
-                    {
-                        logger.debug("Using " + transcriberClass + " as the transcriber class.");
-                    }
+                    logger.info("Using " + transcriberClass + " as the transcriber class.");
+                    conn.disconnect();
+                    return transcriberClass;
+                }
+                else
+                {
+                    logger.warn("Could not retrieve transcriber from remote URL. Response code: " + responseCode);
                 }
                 conn.disconnect();
-                return transcriberClass;
             }
             catch (Exception ex)
             {
                 logger.error("Could not retrieve transcriber from remote URL." + ex);
             }
         }
-
-        return JigasiBundleActivator.getConfigurationService()
+        logger.info("Attempting to retrieve transcriber from local configuration.");
+        String customTranscriptionServiceClass
+                = JigasiBundleActivator.getConfigurationService()
                 .getString(
                         CUSTOM_TRANSCRIPTION_SERVICE_PROP,
                         null);
+        if (customTranscriptionServiceClass != null)
+        {
+            logger.info("Using " + customTranscriptionServiceClass + " as the transcriber class.");
```

Comment:
Maybe leave only the print if there is a tenant enabling non default transcription. 
And please add the callContext to the log as the rest of the code.